aries-site/src/pages/learn/how-to-add-search-and-filter-to-datatable-with-data.mdx

```diff
@@ -0,0 +1,196 @@
+import { Box, Image, Notification } from 'grommet';
+
+## Prerequisites
+
+Let’s get started!
+
+Use this [CodeSandbox template](https://codesandbox.io/s/how-to-add-search-and-filter-to-datatable-with-data-7gxmcy?file=/index.js) to start. The template is populated with a paginated DataTable configured to display SpaceX data.
+
+<Notification
+  status="info"
+  width="large"
+  margin={{ bottom: 'medium' }}
+  message="If you’d like to learn more about how to configure a DataTable with a REST API, check out the guide."
+  actions={[
+    {
+      label: 'Guide TBD',
+      href: '#',
+    },
+  ]}
+/>
+
+## Set up Data provider
+
+In `index.js`, start by importing `Data` from grommet.
+
+Then, remove the `data` property from DataTable and place it as a property on Data. The Data component is now the data provider, controlling which data should be rendered in the DataTable.
```

Comment:
should these comments also be in the codesandbox? to reduce the user having to jump back and forth? even a short comment like `replace this with Data`

Reply:
I was trying to have the majority of "instructions" live in this guide so we don't have to manage comments in multiple places.